### PR TITLE
[kesch] Changing curl dependency on netcdf

### PR DESCRIPTION
**Pull request by victorusu**

### CODE REVIEW

File: easybuild/easyconfigs/n/netCDF/netCDF-4.4.1.1-gmvolf-17.02.eb [hunk 1]
```diff
@@ -18,7 +18,9 @@ postinstallcmds = [('ln -s %(installdir)s/lib64 %(installdir)s/lib')]
 
 builddependencies = [
     ('CMake', '3.7.2', '', True),
-    # ('Doxygen', '1.8.9.1'),
+]
+
+dependencies = [
     ('cURL', '7.52.1'),
```
Comment (jgphpc) [inline easybuild/easyconfigs/n/netCDF/netCDF-4.4.1.1-gmvolf-17.02.eb]: Is it still needed ?
https://github.com/eth-cscs/production/blob/master/easybuild/easyconfigs/n/netCDF/netCDF-4.4.1.1-gmvolf-17.02.eb#L22

Comment (victorusu) [reply easybuild/easyconfigs/n/netCDF/netCDF-4.4.1.1-gmvolf-17.02.eb]: yes

Comment (jgphpc) [reply easybuild/easyconfigs/n/netCDF/netCDF-4.4.1.1-gmvolf-17.02.eb]: I meant `curl` is already a `builddependencies` , see the link to master.
Is it needed as a `dependencies` ?

**Comment by jgphpc** [comment]
needs system in the title

**Comment by victorusu** [comment]
retest this please

**Comment by victorusu** [comment]
retest this please

File: easybuild/easyconfigs/n/netCDF/netCDF-4.4.1.1-gmvolf-17.02.eb (hunk 1 above)
Comment (lucamar) [reply easybuild/easyconfigs/n/netCDF/netCDF-4.4.1.1-gmvolf-17.02.eb]: See http://www.unidata.ucar.edu/software/netcdf/docs/getting_and_building_netcdf.html#build_default and this old post as well: https://www.unidata.ucar.edu/support/help/MailArchives/netcdf/msg09780.html

**Comment by lucamar** [comment]
`libcurl` has been installed on Kesch TDS, therefore the dependency is not needed anymore:
```
$ ls /usr/lib64/ | grep -i libcurl
libcurl.so.4
libcurl.so.4.3.0
```

**Pull request closed without merging**